src/elements/content-explorer/ContentExplorer.js

```diff
@@ -763,8 +760,16 @@ class ContentExplorer extends Component<Props, State> {
         this.unselect();
         item.selected = true;
 
+        // ensure that only selected item is marked as selected in currentCollection
+        const newCollection: Collection = { ...this.state.currentCollection };
```

Comment:
currentCollection is already being pulled out above, can you reuse that reference?